assignments/array-methods.js

```diff
@@ -56,22 +56,40 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(i) {
```

Comment:
this is technically sound, however, i is often used to represent index, not the item.  might confuse others down the road.